docs/dm/assets.md

```diff
@@ -56,4 +64,31 @@ When you drop an asset on the board that has templates associated with it, you'l
 </video>
 
 This can be very handy to reuse some tokens you frequently use to prepopulate things like HP and light sources.
-This will also be key to sharing sets with other players.
+This will also be key to sharing sets with other players (see [downloading files](#downloading-files) above).
+
+### Variants
```

Comment:
One other thing that you didn't mention, is that when using variants,
auras have an extra checkbox that dictates whether that aura is shared among all variants or not

Reply:
I think it's covered in line 85, but I have found some typos anyway so I split it up. Hope it is now better.

Reply:
Oh my bad, missed that